__tests__/main.test.ts

```diff
@@ -106,7 +106,8 @@ describe('Run Github Action', () => {
 
       await run()
       expect(setFailedMock).toHaveBeenCalledWith(
-        `Datadog Synthetics tests failed : criticalErrors: 0, passed: 0, failedNonBlocking: 0, failed: 1, skipped: 0, notFound: 0, timedOut: 0`
+        `Datadog Synthetics tests failed : criticalErrors: 0, passed: 0, failedNonBlocking: 0, failed: 1, skipped: 0, notFound: 0, timedOut: 0\n` +
+          `Results URL: https://app.datadoghq.com/synthetics/explorer/ci?batchResultId=undefined`
```

Comment:
nit: We should update the `emptySummary` used in those tests to have a batch URL